package.json

```diff
@@ -2,68 +2,65 @@
   "scripts": {
-    "build": "npx typescript && grunt",
+    "dev": "webpack --config build/webpack.dev.js --mode development --watch --progress",
+    "start": "webpack serve --config build/webpack.dev.js",
+    "lint": "eslint src/**/*.js test/unit/mocks/*.js test/unit/*.js",
+    "build": "rimraf dist && webpack --config build/webpack.prod.js",
+    "doc": "jsdoc -c build/jsdoc/jsdoc_conf.json",
+    "debug": "iron-mocha --require mochahook",
     "debug-mocha": "iron-node node_modules/mocha/bin/_mocha --require mochahook",
     "test": "mocha test/unit --require mochahook",
     "test-functional": "node test/functional/runTests.js --selenium=remote --app=remote",
-    "dev": "grunt dev",
-    "prepublish": "grunt githooks",
-    "prepublishOnly": "grunt prepublish",
-    "debug": "iron-mocha --require mochahook",
+    "prepack": "npm run build",
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha --require mochahook -- test/"
   },
   "devDependencies": {
+    "@babel/core": "^7.13.8",
+    "@babel/eslint-parser": "^7.13.14",
+    "@babel/preset-env": "^7.13.12",
     "babel": "^5.3.1",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
     "babel-preset-env": "^1.7.0",
-    "babelify": "^6.1.0",
-    "browserify-derequire": "^0.9.6",
-    "bs-html-injector": "^3.0.3",
-    "bundle-collapser": "^1.3.0",
     "chai": "^3.4.1",
     "chai-spies": "^1.0.0",
-    "grunt": "^1.0.4",
-    "grunt-babel": "^6.0.0",
-    "grunt-browser-sync": "^2.2.0",
-    "grunt-browserify": "^5.3.0",
-    "grunt-cli": "^1.3.2",
-    "grunt-contrib-clean": "^2.0.0",
-    "grunt-contrib-connect": "^2.0.0",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-jshint": "^0.11.2",
-    "grunt-contrib-uglify": "^4.0.1",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-exorcise": "^2.1.1",
-    "grunt-ftp-push": "^1.2.1",
-    "grunt-githash": "^0.1.3",
-    "grunt-githooks": "^0.6.0",
-    "grunt-jscs": "^2.5.0",
-    "grunt-jsdoc": "^2.4.0",
-    "grunt-mocha-istanbul": "^5.0.2",
-    "grunt-string-replace": "^1.3.1",
+    "eslint": "^7.23.0",
+    "eslint-webpack-plugin": "^2.5.3",
+    "ftp-deploy": "^2.4.1",
     "ink-docstrap": "^1.3.2",
     "intern": "4.8.7",
     "iron-mocha": "^1.0.3",
     "iron-node": "^3.0.23",
     "istanbul": "^0.4.5",
     "jsdoc": "^3.6.2",
     "jsdom": "^13.2.0",
-    "load-grunt-tasks": "^4.0.0",
     "mocha": "^2.3.4",
+    "replace-in-file": "^6.2.0",
+    "rimraf": "^3.0.2",
     "sinon": "^7.3.2",
-    "time-grunt": "^2.0.0",
+    "string-replace-loader": "^3.0.1",
     "typescript": "^4.1.5",
-    "uglify-js": "^3.6.0",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2",
+    "webpack-merge": "^5.7.3",
     "yargs": "16.0.3"
   },
   "dependencies": {
     "codem-isoboxer": "0.3.6",
     "fast-deep-equal": "2.0.1",
-    "imsc": "^1.0.2",
     "html-entities": "^1.2.1",
+    "imsc": "^1.0.2",
     "localforage": "^1.7.1",
+    "polyjuice": "^2.2.4",
```

Comment:
polyjuice should be added as dev-dep as this module is not needed for player operation. In fact, polyjuice should ideally not appear at all as new dep or dev-dep, since it is not needed after the linter config is migrated.

Reply:
Yes, forgot to remove it also.
Now removed